OrchardCore/Initialize-OrchardCoreSolution/Initialize-OrchardCoreSolution.psm1

```diff
@@ -30,11 +30,11 @@ function Initialize-OrchardCoreSolution
     {
         if ([string]::IsNullOrEmpty($NuGetSource))
         {
-            dotnet new install OrchardCore.ProjectTemplates::1.5.0
+            dotnet new install OrchardCore.ProjectTemplates::1.6.0
```

Comment:
Did you also run this?

Reply:
![image](https://user-images.githubusercontent.com/92299130/235220744-9236a336-def8-4aa2-b357-ddae5cba06e5.png)
Yes for me it seems to be working

Reply:
Great then!